README.md

```diff
@@ -175,16 +175,29 @@ specfile.set_version_and_release('2.1', release='3')
 specfile.set_version_and_release('2.1', preserve_macros=True)
 ```
 
+#### Bumping release
+
+To bump release and add a new changelog entry, we could use the following code:
+
+```python
+from specfile import Specfile
+
+with Specfile("example.spec") as spec:
+    spec.release = str(int(spec.expanded_release) + 1)
+    spec.add_changelog_entry("- Bumped release for test purposes")
+```
+
 #### Changelog
 
 ```python
-# adding a new entry, author is automatically determined
-# (using the same heuristics that rpmdev-packager uses) if possible
-specfile.add_changelog_entry('New upstream release 2.1')
+if not specfile.has_autochangelog:
+    # adding a new entry, author is automatically determined
+    # (using the same heuristics that rpmdev-packager uses) if possible
+    specfile.add_changelog_entry('- New upstream release 2.1')
```

Comment:
`add_changelog_entry()` does nothing if `has_autochangelog` is `True` (documented [here](https://packit.dev/specfile/api/specfile#add_changelog_entry) in the docstring), so this isn't really necessary. But it doesn't hurt to have it here for clarity.